core/controllers/topic_editor.py

```diff
@@ -112,7 +112,7 @@ def get(self, topic_id):
 
         question_summaries, skill_descriptions, next_start_cursor = (
             question_services.get_question_summaries_and_skill_descriptions(
-                constants.NUM_QUESTIONS_PER_PAGE, skill_ids, start_cursor)
+                constants.NUM_QUESTIONS_PER_PAGE * 3, skill_ids, start_cursor)
```

Comment:
Add a comment explaining why you did "constants.NUM_QUESTIONS_PER_PAGE * 3"
Or alternatively, add a new constant NUM_QUESTION_SKILL_LINKS_PER_PAGE or some sort and use that instead!

Reply:
Done.